### DISCUSSION TITLE
Documentações solicitadas por pagamento via nota de empenho - Python Cerrado

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#135] [Geral]
Boa tarde! Tudo bem?

Poderiam nos ajudar com uma questão de nota de empenho?

A Python Cerrado recebeu uma solicitação de pagamento via nota de empenho, e precisamos enviar as seguintes documentações para o processo:

- [x] 1.Comprovante de inscrição do CNPJ 
- [x] 2. Certidão Negativa da Receita Federal Certidão de regularidade do FGTS 
- [x] 3. Certidão Negativa de Débitos Trabalhistas
- [ ] 4. Link de Acesso Formulário de Ciência da Contratação Direta
- [ ] 5. Link de Acesso Declaração de Ausência de Nepotismo
- [ ] 6. Link de Acesso Declaração de Inexistência de Vínculo com o TJDFT
- [x] 7. Link de Acesso Declaração de optante pelo Simples Nacional, se inscrita
- [x] 8. Link de Acesso Estatuto Social da empresa, procuração, ou instrumento equivalente, para comprovação de que o signatário dos documentos acima possui poderes para representar a empresa - https://python.org.br/estatuto/

cc @cecivieira

**Comment by cecivieira** [comment]
@anicelysantos quando a declaração for assinada, coloca aqui, por favor
ALém disso, coloca dentro da pasta onde esses documentos estão sendo reunidos

**Reply by anicelysantos** [reply]
todos documentos assinados. Ao invés de colocar aqui, vou dar acesso a pasta pra tu @anadulce

**Reply by anicelysantos** [reply]
aaah Ana já tem acesso... seguimos, então! :)

**Comment by cecivieira** [comment]
@tiidadavena tá reunindo a documentação solicitada

**Comment by tiidadavena** [comment]
> Boa tarde! Tudo bem?
> 
> Poderiam nos ajudar com uma questão de nota de empenho?
> 
> A Python Cerrado recebeu uma solicitação de pagamento via nota de empenho, e precisamos enviar as seguintes documentações para o processo:
> 
>     * [x]  1.Comprovante de inscrição do CNPJ
> 
>     * [x]  2. Certidão Negativa da Receita Federal Certidão de regularidade do FGTS
> 
>     * [x]  3. Certidão Negativa de Débitos Trabalhistas
> 
>     * [ ]  4. Link de Acesso Formulário de Ciência da Contratação Direta
> 
>     * [ ]  5. Link de Acesso Declaração de Ausência de Nepotismo
> 
>     * [ ]  6. Link de Acesso Declaração de Inexistência de Vínculo com o TJDFT
> 
>     * [ ]  7. Link de Acesso Declaração de optante pelo Simples Nacional, se inscrita
> 
>     * [x]  8. Link de Acesso Estatuto Social da empresa, procuração, ou instrumento equivalente, para comprovação de que o signatário dos documentos acima possui poderes para representar a empresa - https://python.org.br/estatuto/
> 
> 
> cc @cecivieira

@anadulce qual o nome da instituição que solicitou as documentações?

**Reply by tiidadavena** [reply]
Também precisarei da info se será por Inexigibilidade ou por Dispensa de Licitação

**Reply by tiidadavena** [reply]
Acredito que será por inexigibilidade, mas precisaremos dos seguintes dados da instituição para gerar os documentos tendo em vista que somos apenas uma ONG, não tendo natureza de produção de bens e serviços 🫠 🥺

**Reply by tiidadavena** [reply]
- Razão social da instituição
- CNPJ da instituição
- No do pregão eletrônico (caso aplicável)

**Reply by anadulce** [reply]
Boa tarde, @tiidadavena <3
Muito obrigada informações, já foram solicitadas!

**Comment by tiidadavena** [comment]
cc @anadulce @cecivieira @anicelysantos  
Os documentos 4, 5 e 6 precisam tanto dos dados da instiuição solicitante quanto da assinatura da Presidência da APyB.

**Comment by cecivieira** [comment]
@anadulce coleta os dados da empresa, por favor

**Reply by anadulce** [reply]
Os dados da empresa foram solicitados, retorno assim que recebê-los.

**Comment by cecivieira** [comment]
Coloca o link da pasta aqui

**Reply by anicelysantos** [reply]
https://drive.google.com/drive/folders/1VP05j2phMGtt-cLN9HI5kPjndWJjbjMH?usp=sharing

**Reply by anadulce** [reply]
Muito obrigadaaaaa <3

**Reply by belaaiza** [reply]
@cecivieira consege compartilhar comigo?

**Comment by cecivieira** [comment]
@belaaiza já liberei acesso para você na pasta, mas estou revisando a validade das certidões. Te aviso quando terminar

**Reply by cecivieira** [reply]
@belaaiza conclui a revisão. Todos os certificados da pasta estão com validade atual.
